contracts/FlowStakingCollection.cdc

```diff
@@ -749,8 +855,46 @@ pub contract FlowStakingCollection {
             return delegatorInfo
         }
 
+        /// Function to get the list of all of the machine accounts for a node in the collection
+        /// Key: Node ID
+        /// Value: Account address for the account that stores the QC or dkg object
+        pub fun getMachineAccounts(): {String: Address} {
+            let accountsRecord = FlowStakingCollection.account.copy<{String: Address}>(from: /storage/epochMachineAccounts)
```

Comment:
If I'm understanding correctly, the machine accounts are stored in the storage of the account where the `FlowStakingCollection` contract is deployed (presumably the service account) rather than in the user's own account. I'm curious why it's stored there rather than in the user's own account. Seems like a potential abuse vector if we provide a mechanism to store data where the storer doesn't have to pay for it 🤔 .

Reply:
Like with my other comment, this is an upgrade to the original staking collection in this PR: https://github.com/onflow/flow-core-contracts/pull/119

So we can't create that field in this version because adding fields in upgrades isn't allowed. Should I include it in the first version anyway even though we won't use it?